tasty/src/dotty/tools/tasty/TastyFormat.scala

```diff
@@ -474,6 +475,7 @@ object TastyFormat {
   final val INFIX = 43
   final val EMPTYCLAUSE = 44
   final val SPLITCLAUSE = 45
+  final val INVISIBLE = 46
```

Comment:
`isLegalTag` in this file still says:
```scala
firstSimpleTreeTag <= tag && tag <= SPLITCLAUSE ||
```
presumably that should be replaced with `INVISIBLE` (we should really add something like `final val LAST_SIMPLE_TAG = INVISIBLE` to avoid having to update multiple places like this).